lib/config/manifest.json

```diff
@@ -1,10 +1,11 @@
 {
   "maintainer": "help@datadoghq.com",
-  "manifest_version": "0.1.0",
+  "manifest_version": "0.1.2",
```

Comment:
It is my understanding that `manifest_version` was nothing more than a version for the manifest in question and constrained to each check. Did you intend to use it as a variable to track iterations of the complete repo?